movieapp/src/pages/HomePage/HomePage.tsx

```diff
@@ -2,12 +2,20 @@ import { Navigation } from '../../components/Navigation/Navigation';
 import { SortElement } from '../../components/SortElement/SortElement';
 import { MovieList } from '../../containers/MovieList/MovieList';
 
-export function HomePage(): JSX.Element {
+type MovieState = {
+    setMovies: Function;
+    movies: [];
+}
+
+export function HomePage(props:MovieState): JSX.Element {
   return (
     <>
-      <Navigation />
-      <SortElement />
-      <MovieList />
+      <Navigation key='navigation'  />
```

Comment:
too many spacese before />  such things should be handled by eslint

Reply:
fixed